lib/resources/kernel_module.rb

```diff
@@ -37,13 +66,56 @@ def loaded?
       !found.nil?
     end
 
+    # @note maintain logcal agreement with 'disabled?' method
+    alias enabled? loaded?
+
+    def blacklisted?
+      if inspec.os.redhat? || inspec.os.name == 'fedora'
+        modprobe_cmd = "/sbin/modprobe --showconfig | grep blacklist | grep #{@module}"
+      else
+        modprobe_cmd = "modprobe --showconfig | grep blacklist | grep #{@module}"
+      end
+      cmd = inspec.command(modprobe_cmd)
+      cmd.exit_status.zero? ? cmd.stdout.delete("\n") : nil
```

Comment:
I think you're misunderstanding. The method name is fine and expected. The ? character in the last line is the Ruby ternary operator and, as written, will not return true or false as is expected for a `be_` matcher or a method that is expected to return Boolean. This is what needs to be corrected.